kubernetes-client/pom.xml

```diff
@@ -249,6 +249,7 @@
     <dependency>
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
```

Comment:
nit: this is redundant, the scope is inherited from the `<dependencyManagement>` section in the parent pom (just like the version).